### DISCUSSION TITLE
Error during table creation

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3621] [Q&A]
Hi,

While creating table I am getting below error

Access to XMLHttpRequest at 'https://example.com/api/v1/db/meta/projects/p_yen5b6389lbuei/tables' from origin 'https://example.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
688b038.js:2  POST https://example.com/api/v1/db/meta/projects/p_yen5b6389lbuei/tables net::ERR_FAILED 504

 Can you suggest how to solve this issue?

Thanks and Regards,
Sumathy Venugopal

**Comment by o1lab** [comment]
Looks like you are running behind a reverse proxy. If so, you will have to figure the cors part. I've seen several customers successfully able to do so without a problem/

**Reply by MGW-SumathyV** [reply]
We had DB issue and its resolved now
